02 ReactApp/src/pods/organization/component.js

```diff
@@ -0,0 +1,28 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+
+export class OrganizationComponent extends Component {
+
+  componentDidMount() {
+    this.props.fetchMembers();
+  }
+
+  render() {    
+    return (
```

Comment:
Also I would do `PureComponent` instead of `Component` if you are using class notation instead of stateless functions.